<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-43J3MGE8" failures="1" tests="1" name="Regression Test" time="8.824" errors="0" timestamp="2023-03-31T20:15:22 IST">
  <testcase classname="com.ds.testLayer.LoginPageTest" name="verifyLoginTest" time="3.368">
    <failure type="java.lang.AssertionError" message="expected [https://www.apps.dalalstreet.ai/dashboard] but found [https://www.apps.dalalstreet.ai/login]">
      <![CDATA[java.lang.AssertionError: expected [https://www.apps.dalalstreet.ai/dashboard] but found [https://www.apps.dalalstreet.ai/login]
at com.ds.testLayer.LoginPageTest.verifyLoginTest(LoginPageTest.java:24)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verifyLoginTest -->
</testsuite> <!-- Regression Test -->
